cmd/server.go

```diff
@@ -20,7 +20,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	homedir "github.com/mitchellh/go-homedir"
+	"github.com/mitchellh/go-homedir"
```

Comment:
Why did you remove `homedir`? It is referenced later in the file, on line 546 https://github.com/cloudposse/atlantis/blob/60e506dca69dc3c8aba1a41562da4fcf5600c1e7/cmd/server.go#L546

Reply:
those aliases are redundant and were removed by the IDE
the code compiles with and without them